core/App/navigators/RootStack.tsx

```diff
@@ -104,7 +104,14 @@ const RootStack: React.FC = () => {
     return (
       <Stack.Navigator initialRouteName={Screens.Splash} screenOptions={{ ...defaultStackOptions, headerShown: false }}>
         <Stack.Screen name={Screens.Splash} component={splash} />
-        <Stack.Screen name={Screens.EnterPin}>
+        <Stack.Screen
+          name={Screens.EnterPin}
+          options={() => ({
+            headerShown: true,
+            headerLeft: () => false,
+            rightLeft: () => false,
```

Comment:
Can we make sure that we can translate the header with: `title: t(..)`

Reply:
Fixed